source/Assets/Plugins/Didomi/Editor/DidomiPaths.cs

```diff
@@ -0,0 +1,29 @@
+using System.IO;
+using UnityEngine;
+
+/// <summary>
+/// Properties and methods related to paths used to handle Didomi dependencies.
+/// </summary>
+public static class DidomiPaths
```

Comment:
The file name should probably be changed, as it is not only related to paths, and is specific to iOS currently. Maybe something like `DidomiIOSDependencies`?

Reply:
I'll be pushing an update that will change the references to this class from things like `DidomiPaths.FRAMEWORK_URL` to `DidomiPaths.IOS.FRAMEWORK_URL`. That way we can keep properties that are common to all platforms still under DidomiPaths, for example `DidomiPaths.PACKAGE_JSON_PATH` and the Android specific Paths under `DidomiPaths.Android`.